### PR TITLE
docs(cli): Add missing space in --help output

### PR DESCRIPTION
**Pull request by rpatterson**

### CODE REVIEW

**Comment by rpatterson** [comment]
I haven't tested this but it should work.  I don't have much time to follow up on this, so do with it what you will.  Thanks for the useful tool!

File: sphinxlint/__main__.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def job_count(values):
         action=StoreNumJobsAction,
         help="Run in parallel with N processes. Defaults to 'auto', "
         "which sets N to the number of logical CPUs."
```
Comment (ezio-melotti) [inline sphinxlint/__main__.py:128]:
```suggestion
        "which sets N to the number of logical CPUs. "
```

File: sphinxlint/__main__.py [hunk 2]
```diff
@@ -126,7 +126,7 @@ def job_count(values):
...
-        "Values <= 1 are all considered 1.",
+        " Values <= 1 are all considered 1.",
```
Comment (ezio-melotti) [inline sphinxlint/__main__.py:129]:
```suggestion
        "Values <= 1 are all considered 1.",
```

**Review by ezio-melotti — APPROVED** [review]
Thanks for the report! For consistency I moved the missing space at the end of the previous strung.

**Pull request merged**